config.xml

```diff
@@ -20,6 +20,7 @@
     <platform name="android">
         <hook src="attach-source-maps.js" type="before_build" />
         <preference name="android-targetSdkVersion" value="32" />
+        <preference name="AndroidLargeHeap" value="true" />
```

Comment:
@vishwanath1004  largeHeap = true is already there in manifest . SO you can remove it.